tests/unit/components/HdAlert.spec.js

```diff
@@ -1,54 +1,60 @@
-import { shallowMount } from '@vue/test-utils';
+import { wrapperFactoryBuilder } from 'tests/unit/helpers';
 import HdAlert from '@/components/HdAlert.vue';
 
+const defaultSlot = '<b>Default slot</b>';
+
+const wrapperBuilder = wrapperFactoryBuilder(HdAlert, {
+  slots: {
+    default: defaultSlot,
+  },
+  shallow: true,
+});
+
 const MESSAGE_SELECTOR = '.alert__message';
 const ICON_SELECTOR = '.alert__icon';
 
-const CONTENT = '<b>Default slot</b>';
-
 describe('HdAlert', () => {
-  let wrapper;
+  it('renders component', () => {
+    const wrapper = wrapperBuilder();
 
-  beforeEach(() => {
-    wrapper = shallowMount(HdAlert, {
-      slots: {
-        default: CONTENT,
-      },
-    });
-  });
-
-  test('renders component', () => {
     expect(wrapper.html()).toMatchSnapshot();
   });
 
-  test('renders the slot correctly', () => {
-    expect(wrapper.find(MESSAGE_SELECTOR).element.innerHTML).toBe(CONTENT);
+  it('renders the slot correctly', () => {
+    const wrapper = wrapperBuilder();
+
+    expect(wrapper.find(MESSAGE_SELECTOR).element.innerHTML).toBe(defaultSlot);
   });
 
-  test('renders the right type', () => {
-    // It doesn't make sense to go through all the possible types (we may have many)
-    // they share the same logic, so if one works the rest should work as well
+  it('renders the right type', () => {
     const TEST_TYPE = 'warning';
-
-    wrapper.setProps({
-      type: TEST_TYPE,
+    const wrapper = wrapperBuilder({
+      props: {
+        type: TEST_TYPE,
+      },
     });
+    // It doesn't make sense to go through all the possible types (we may have many)
```

Comment:
Maybe we could have an array of types, and assert that every type is working correctly?

Reply:
I will update this test to match: https://github.com/homeday-de/homeday-blocks/blob/develop/tests/unit/components/HdLink.spec.js#L21